src/openrct2/peep/peep.h

```diff
@@ -685,16 +685,16 @@ enum {
  * applied in tautology if statements.
  */
 #define FOR_ALL_PEEPS(sprite_index, peep) \
-    for (sprite_index = gSpriteListHead[SPRITE_LIST_PEEP]; sprite_index != SPRITE_INDEX_NULL; sprite_index = peep->next) \
-        if ((peep = GET_PEEP(sprite_index)) != NULL || 1)
+    for ((sprite_index) = gSpriteListHead[SPRITE_LIST_PEEP]; (sprite_index) != SPRITE_INDEX_NULL; (sprite_index) = peep->next) \
```

Comment:
Why did you surround this with quotes?

Reply:
You should always place brackets around macro-parameters to avoid a category of really annoying bugs. See https://docs.freebsd.org/info/cpp/cpp.info.Macro_Parentheses.html

Reply:
Agreed it is the best thing to do. You missed a sprite index on the next line.

Reply:
No, in the next line it's not needed as GET_PEEP already has one :)